force-app/main/default/classes/SOQL.cls

```diff
@@ -174,7 +235,8 @@ public with sharing class SOQL {
      * @return A list of the first 10 Accounts ordered by Name.
      */
     public static List<Account> getFirstTenAccounts() {
-        return null; // Replace null with the variable you used to store the result
+        List<Account> firstTenAccts = [SELECT ID, Name FROM Account ORDER BY Name LIMIT 10];
+        return firstTenAccts; // Replace null with the variable you used to store the result
```

Comment:
Try to combine these two into one line

Reply:
Done - But is this best practice, my assumption is that the variable is better ! Am I correct